### DISCUSSION TITLE
GroupPadding

### DISCUSSION THREAD

**Discussion by ptasznikarium** [Live-Charts/LiveCharts2#1541] [Q&A]
It seems docs are out of date. 
https://livecharts.dev/docs/WPF/2.0.0-rc2/CartesianChart.Column%20series%20properties#grouppadding-property

I see that GroupPadding is obsolete and in fact it is doing nothing for me in ColumnSeries.
Tooltip says to replace it by Padding, but this is doing something very different.

Is there any way to set spacing between groups?

![image](https://github.com/user-attachments/assets/b127921c-3605-4a0a-b9ae-bd235a68dad4)

**Comment by mario-dg** [comment]
Also heavily requesting to bring that feature back.
